packages/manager/modules/pci/src/projects/project/data-processing/submit-job/submit-job.controller.js

```diff
@@ -2,11 +2,18 @@ import { find } from 'lodash';
 
 export default class {
   /* @ngInject */
-  constructor($scope, $state, dataProcessingService, CucRegionService) {
+  constructor(
+    $scope,
+    $state,
```

Comment:
`$state` doesn't seem used and we should check the need for `$scope` here (https://github.com/ovh/manager/blob/365be6d5832bca2c7676410131d87074791df8b2/packages/manager/modules/pci/src/projects/project/data-processing/submit-job/submit-job.controller.js#L218) but it is out of scope of this PR